ExampleMod/Content/Items/Tools/ExamplePickaxe.cs

```diff
@@ -38,6 +40,14 @@ public class ExamplePickaxe : ModItem
 			}
 		}
 
+		public override void UseAnimation(Player player) {
+			// Randomly causes the player to use Example Pickaxe Emote when using the item
+			if (Main.rand.NextBool(20)) {
+				// Note that we don't have to sync the emote because UseAnimation runs on all clients and server
+				EmoteBubble.MakePlayerEmote(player, ModContent.EmoteBubbleType<ExamplePickaxeEmote>(), false);
```

Comment:
The value of `Main.rand` will not be the same on all clients, so you should only do this on one machine and sync the emote so all players see it at the same time